admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue

```diff
@@ -247,6 +247,13 @@
             </div>
         </div>
     </div>
+
+      <admin-components-publishingmodal
+          v-if="publishDialogPath"
+          :isOpen="publishDialogPath"
+          :path="publishDialogPath"
+          @complete="closePublishing"
```

Comment:
please move `@complete` below `:modalTItle`

(order is: first `v-bind/:` then `v-on/@`)

Reply:
done